3D Chess/Packages/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "dependencies": {
+    "com.unity.collab-proxy": "1.5.7",
```

Comment:
Are all these packages needed? We can probably remove most of them.

Reply:
It's a standard Unity 3D template. I've left everything the way it is to make sure if a user want to follow along and creates a diff of their repo and this to only see differences as expected.
For the same reasons I kept things like ParalleSync in their default location and leave other things as default as possible (for examples at least).
What's your take on that?

Reply:
Yeah, I have mixed feelings - on one hand, it's good not to stray too far from the template, on the other I've always found it annoying to download some super bloated example that pulls half the universe to show me how to call a simple API. I guess I'm fine with keeping the packages if you feel it's going to be easier for readers to follow along, but I'd definitely move ParrelSync to UPM to get rid of its sources from the example.

Reply:
Yup, done. See above.